examples/docker/census-on-omnisci/census-omnisci.dockerfile

```diff
@@ -49,16 +46,12 @@ RUN conda update -n base -c defaults conda -y && \
     conda config --set channel_priority strict && \
     conda create -n modin --yes --no-default-packages && \
     conda activate modin && \
-    conda install -c intel/label/modin -c conda-forge modin "ray>=1.0.0" "numpy<1.20.0"
-
-RUN conda activate modin && \
-    conda install -c intel/label/modin -c conda-forge -c intel ${add_extra_channel} \
-        "daal4py>=2021.1" dpcpp_cpp_rt && \
-    conda install -c conda-forge "scikit-learn<0.24.0" && \
+    conda install -c intel/label/modin -c conda-forge -c intel \
+        modin "ray>=1.0.0" "numpy<1.20.0" "scikit-learn<0.24.0" "daal4py>=2021.1" dpcpp_cpp_rt && \
     conda clean --all --yes
 
-COPY ipums_education2income_1970-2010.csv "${HOME}/ipums_education2income_1970-2010.csv"
-
 COPY census-omnisci.py "${HOME}/census-omnisci.py"
+RUN mkdir /dataset
+RUN echo 'conda activate modin && python -u ${HOME}/census-omnisci.py $*' > /entrypoint.sh
```

Comment:
Have you tested passing parameters into `census-omnisci.py`?

Reply:
Yes I certainly did. This script no longer works when parameters are not passed.